packages/backend/.eslintrc.json

```diff
@@ -3,6 +3,7 @@
     // Required for certain syntax usages
     "ecmaVersion": 2018
   },
+  "parser": "babel-eslint",
```

Comment:
Adding this parser might make you able to write babel code with no complaint by ESLint but it won't compile when we deploy the app because Firebase only understand commonJS syntax. Please remove this parser and edit any code to adapt commonJS syntax instead of ES6 (babel)

Reply:
Ok, forgot about that